apps/barista/src/layout/nav/nav.html

```diff
@@ -15,7 +15,11 @@
   </div>
   <nav id="ba-nav-bar" class="ba-nav-bar">
     <ul class="ba-nav-list">
-      <li class="ba-nav-item" *ngFor="let item of navItems">
+      <li
+        class="ba-nav-item"
+        *ngFor="let item of (_navData$ | async)?.navItems"
```

Comment:
can we create an own nav component for the item that get the data via an input so that we don't use the async pipe twice?

<ba-nav [data]="_navData$ | async"><ba-nav>

Reply:
Where do you see a second async for `_navData$`?